src/Controller/OcrController.php

```diff
@@ -193,6 +201,15 @@ private function getApiResponse(array $params): JsonResponse
         $response->setEncodingOptions(JSON_NUMERIC_CHECK);
         $response->setStatusCode(Response::HTTP_OK);
 
+        // Expose flash messages.
+        /** @var FlashBag $flashBag */
+        $flashBag = $this->session->getBag('flashes');
+        // @phan-suppress-next-line PhanUndeclaredMethod
```

Comment:
(FTR, you can avoid the double suppression by adding

```
'@phan-var FlashBag $flashBag';
```

at line 207, as a string literal).

Reply:
Nice! I did not know about that, done. A pity that Phan can't read comments!

Reply:
Yeah, unfortunately that's a hard limitation. Inline comments are stripped by the php-ast extension when creating the AST, so phan doesn't even know that they exist.